sonar-scanner-engine/src/main/java/org/sonar/scanner/bootstrap/GlobalContainer.java

```diff
@@ -85,7 +85,8 @@ private void addBootstrapComponents() {
       new SonarQubeVersion(apiVersion),
       SonarRuntimeImpl.forSonarQube(apiVersion, SonarQubeSide.SCANNER),
       StoragesManager.class,
-      GlobalSettings.class,
+      MutableGlobalSettings.class,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "MutableGlobalSettings"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)